main.go

```diff
@@ -93,6 +84,25 @@ func getMapKeys(m map[string]interface{}) []string {
 	return keys
 }
 
+func resourceToStateKeyStr(path []string, resourceKey string) (string, error) {
+	if len(path) < 1 {
+		return "", errors.New("no modules found in path given to resourceToStateKeyStr")
+	}
+
+	if len(path) == 1 && path[0] == "root" {
+		return resourceKey, nil
+	}
+	s := "module." + strings.Join(path[1:], ".module.") + "." + resourceKey
```

Comment:
I believe 'path[1:]' drops the first char, it that is your intent

Reply:
yes, that is the intent.

good point.